gammapy/background/fov.py

```diff
@@ -3,3 +3,60 @@
 Field-of-view (FOV) background estimation
 """
 from __future__ import print_function, division
+from astropy.wcs import WCS
+from astropy.wcs.utils import pixel_to_skycoord
+import numpy as np
+from scipy.interpolate import interp1d
+
+__all__ = ['fill_acceptance_image',
+           ]
+
+
+def fill_acceptance_image(image, center, offset, acceptance):
+    """Generate a 2D image of a radial acceptance curve.
+
+    The radial acceptance curve is given as an array of values
+    defined at the specified offsets.
+
+    Parameters
+    ----------
+    image : `~astropy.io.fits.ImageHDU`
+        Empty image to fill.
+    center : `~astropy.coordinates.SkyCoord`
+        Coordinate of the center of the image.
+    offset : `~numpy.ndarray` of `~astropy.coordinates.Angle`
+        Array of offset values in degrees where acceptance is defined.
+    acceptance : `~numpy.ndarray`
+        Array of acceptance values.
+
+    Returns
+    -------
+    image : `~astropy.io.fits.ImageHDU`
+        Image filled with radial acceptance.
+    """
+    #initialize WCS to the header of the image
```

Comment:
There should be a space after `#`, i.e. the comment is
```
# initialize WCS
```
Discussing code formatting during code review is tedious for both parties.
That's why the Python programmers agreed on a style guide in [PEP 8](https://www.python.org/dev/peps/pep-0008/#comments) and developed tools (`pep8` and `autopep8`) to check and auto-format their code.

Please do this for all the code you write (after a while it'll be correctly formatted without you thinking about it).
But avoid mixing formatting fixes for the whole file with your code changes.
I do this by selecting the function I wrote in Pycharm and selecting "format code".
I'm not sure which tool it runs in the background, but it does a good job.